pkg/service/v1alpha1/node.go

```diff
@@ -119,29 +120,43 @@ verifyPublish:
 		// which implies that either the mount operation is complete
 		// or under progress.
 		// Lets verify if the mount is already completed
-		if info.Spec.Volume.DevicePath != "" {
+		if info.Spec.Volume.MountPath != mountPath {
```

Comment:
functions became big.. better to split them

Reply:
split this to make it more manageable